pkg/osquery/tables/crowdstrike/falconctl/table.go

```diff
@@ -89,13 +90,26 @@ OUTER:
 		output, err := t.execFunc(ctx, t.logger, 30, falconctlPaths, args, false)
 		if err != nil {
 			level.Info(t.logger).Log("msg", "exec failed", "err", err)
-			return nil, err
+			synthisized_data := map[string]string{
```

Comment:
Typo -- I think we probably want `synthesizedData` for consistent formatting

Reply:
hahahaha. You're totally right. I've been writing too much ruby.